### PR TITLE
Install reticulate from Debian.

### LINKED ISSUE #60: Speed up R install
Currently the R install requires about 5 minutes. It would be nice if we could speed this up by using r-cran-x packages from Debian rather than compiling them each time. I had a quick go at this, but it didn't work for me.

### PR DESCRIPTION
**Pull request by jeromekelleher**
Try to resolve #60 and #61

### CODE REVIEW

**Comment by jeromekelleher** [comment]
This is weird. Looks like we have some non-repo version of R installed or something. I don't understand why the cran packages are broken, but it looks like this is a fairly common problem.

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -33,8 +33,9 @@ jobs:
 
     - name: Install R support
       run: |
-        sudo apt-get install r-base-dev
-        sudo R -e 'install.packages(c("reticulate", "IRkernel"))'
+        sudo apt-get remove r-base
```
Comment (brieuclehmann) [inline .github/workflows/build.yml:36]: Changing this to `sudo apt-get remove r-base-core` worked for me, though it still took c. 4min to build the whole thing. Weirdly the r-base-core version already installed is v4.0.5 which is incompatible with the r-cran-reticulate version that gets installed here. But when you reinstall it (via the r-cran-reticulate dependency), it installs v3.6.3 which is apparently OK. See [build](https://github.com/brieuclehmann/tutorials/runs/2479369702).

**Review by brieuclehmann — COMMENTED** [review]
Remove `r-base-core` seems to work though it still takes about 4min to build. Another option would be to `sudo apt-add-repository` so that a version of `r-cran-reticulate` that is compatible with R v4.0.5 gets installed but I haven't been able to figure out how to do this.

**Comment by jeromekelleher** [comment]
Great, looks like this is working and the install is down to 2 minutes. Thanks @brieuclehmann!

**Pull request merged**